File: prob2/prob2.py
#!/usr/bin/env python

sumOf = 0

#The two fib numbers
x = 1
y = 1

stopAt = 4000000

while(y<stopAt):
    print("y is: ", y)
    if y%2==0:
        sumOf+=y
    i = y #remeber the value of y
    y = x + y
    x = i

print(sumOf)

    

File: prob2/prob2.c
#include<stdio.h>

void main(){
  int sumOf=0;
  int x = 1;
  int y = 1;
  int stopAt = 4000000;

  while(y<stopAt){
    if(y%2==0) sumOf+=y;
    int i = y;
    y = x + y;
    x = i;
  }

  printf("%d\n", sumOf);
}
